fern/definition/medical/fhir.yml

```diff
@@ -8,7 +8,23 @@ service:
   endpoints:
     startConsolidatedQuery:
       display-name: Start Consolidated Data Query
-      docs: Trigger a consolidated data query for the given patient
+      docs: [
+          Trigger a consolidated data query for the given patient.
+          When executed,
+          this endpoint triggers an asynchronous query to retrieve a Patient's consolidated data from the FHIR repository.
+          Once the data is consolidated and ready for consumption,
+          a webhook request is sent to your configured URL containing the Patient's
+          data in FHIR-compliant format,
+          based on the parameters used to trigger the query.
+          You'll be able to query multiple times for the same Patient which will be added to the queue and
+          processed in order. If the parameters are the same as a previous query that is still processing,
+          the new query will be ignored.
+          You can check the status of the data query by calling get consolidated data query.
+          This endpoint also provides the ability to render a Medical Record Summary
+          from the FHIR bundle as a PDF or HTML document. This will be returned via a webhook
+          as a FHIR bundle with a DocumentReference resource
+          that will contain a URL to retrieve the data from.,
```

Comment:
Isn't it missing the section explaining what it returns?

Reply:
Isnt that in that in the response: docs: section?